cookbooks/aws-parallelcluster-computefleet/templates/jobinfomgtd/parallelcluster_jobinfomgtd.conf.erb

```diff
@@ -0,0 +1,7 @@
+[jobinfomgtd]
+cluster_name = <%= node['cluster']['stack_name'] %>
```

Comment:
I think 'cluster_name' is preferred to 'stack_name', but both are in use for the head node, so either way is fine.